pkg/operator/apiserver/audit/testdata/allrequestbodies.yaml

```diff
@@ -8,8 +8,10 @@ omitManagedFields: true
 omitStages:
 - "RequestReceived"
 rules:
-# Don't log requests for events
-- level: None
+# Log events at the metadata level. This is necessary to identify misconfigured
+# clients. They could either spam mutating requests on Events or list all the
+# Events in the cluster which are two operations that can lead to disruption.
+- level: Metadata
```

Comment:
I would also remove it. If a user requested all request bodies, let's give them what they want.